### PR TITLE
feat added: Button's CSS added

### PR DESCRIPTION
**Pull request by rdshinde**
Screenshot 2022-01-31 at 6 37 48 PM

Screenshot 2022-01-31 at 6 38 17 PM

Screenshot 2022-01-31 at 6 38 43 PM

### CODE REVIEW

**Review by shilpe26 — COMMENTED** [review]
The code is all good. 
You can proceed with merging it👍🏻

File: stylesheets/utils/utility.css [hunk 1]
```diff
@@ -68,4 +68,7 @@ img {
   max-width: 100%;
   height: auto;
 }
+.shadow-lg{
+  box-shadow: #00000059 0px 5px 15px;
+}
```
Comment (Mohasin-Haque) [inline stylesheets/utils/utility.css:74]: You can make variable for box -shadow as well. Rest your code is so clean and tidy.. Good to go!!

**Review by Mohasin-Haque — COMMENTED** [review]
Looks good .. Work on the commented things.

File: stylesheets/utils/utility.css (hunk 1 above)
Comment (rdshinde) [reply stylesheets/utils/utility.css:74]: Sure.

**Pull request merged**